package.json

```diff
@@ -42,22 +43,22 @@
     "hyphenate-style-name": "^1.0.3"
   },
   "devDependencies": {
-    "@babel/cli": "^7.2.0",
-    "@babel/core": "^7.2.2",
-    "@babel/preset-env": "^7.2.0",
-    "babel-core": "7.0.0-bridge.0",
-    "babel-eslint": "^7.1.1",
-    "babel-jest": "^23.6.0",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.24.1",
-    "cross-env": "^5.2.0",
-    "eslint": "^3.14.1",
-    "eslint-config-airbnb": "^14.0.0",
-    "eslint-plugin-flowtype": "^2.30.0",
-    "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^3.0.2",
-    "eslint-plugin-react": "^6.9.0",
-    "jest": "^23.6.0",
+    "@babel/cli": "^7.8.4",
+    "@babel/core": "^7.9.0",
+    "@babel/preset-env": "^7.9.5",
+    "@babel/preset-typescript": "^7.9.0",
+    "@types/jest": "^25.2.1",
+    "@typescript-eslint/eslint-plugin": "^2.26.0",
+    "@typescript-eslint/parser": "^2.26.0",
+    "babel-jest": "^25.2.6",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
+    "cross-env": "^7.0.2",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "jest": "^25.2.6",
     "prettier": "^1.7.4",
```

Comment:
ESLint & Babel were updated for the best experience with the TypeScript conversion (Babel was recent so that upgrade was painless; ESLint, though, needed to be updated to work properly with TypeScript since it added official support).

I didn’t update Prettier in this PR, but happy to update that too if desired—that should be minimal.

Reply:
Likewise, Jest was updated since it, too, added first-class TypeScript support in recent versions. Tests still pass, and no other configuration changed.